### ISSUE TITLE
Unable to use scp without password prompt by default

### ISSUE THREAD

**Issue by RHEmployee** [psss/tmt#570]
As a 1minutetip user, I'm used to have ~/.ssh/config configured by official guideline, and copy files to remote machine via scp/rsync/whatever without password prompt.
Provided configuration involves that keys are used automatically with 1minutetip instances so I should not bother with password prompt.

How to reproduce:

On first terminal:
$ tmt run -vvvddd provision -h minute -i rhel-8 login finish 
...
Run command 'ssh -oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null -i /usr/share/qa-tools/1minutetip/1minutetip root@10.0.205.68 whoami'.
...

on second terminal:
ssh root@10.0.205.68
root@10.0.205.68's password:

scp have same result.

Yes as a workaround I can use provided ssh key but it's much more user friendly to have ~/.ssh/config
configured so it can do it automatically.

**Comment by psss** [comment]
Thanks for the issue. Just tried on my laptop and the scenario described above works for me without the need to provide a password. I have the 1mt key enabled in my ssh config. Are there any special 1mt instructions how to enable the key?

**Comment by RHEmployee** [comment]
So you were lucky or you have different ssh_config.
It just doesn't work from scrach on fedora 33 with settings below which are recommended.

More info:
https://internalwiki/BaseOs/Projects/1minuteTIP#Connecttoreservedtomachineviassh

**Comment by FrNecas** [comment]
I've looked into the issue (using the provided ssh config) and managed to reproduce it. Looks like the IP of the provisioned machine doesn't match any `Host` in the config and hence the SSH key isn't passed. If I had to guess, it is caused by using a different network than 1minutetip (related to #555 ). So if we wanted to be compatible with 1minutetip, we would have to hardcode the one specific network... Lets discuss this one at tmt hacking?
